src/components/inputs/localized-money-input/localized-money-input.percy.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {
```

Comment:
can you refactor this to match our current VRT tests? 

we need one .visualroute.js and one .visualspec.js 👍